Implementacija/savetovaliste/app/Views/pregled_pitanja.php

```diff
@@ -1,154 +1,179 @@
                         <div class="input letters_dark_blue">
-                            <p style="text-align: justify; font-weight: normal;">Anksioznost je problem koji me muči već godinama. Pokušavala sam na razne načine da ga rešim, išla sam kod psihologa i psihijatra, ali nije bilo većih poboljšanja. Možda zato što nisam bila spremna da otvoreno govorim sa njima. Strah me je da podelim priču sa bliskim osobama, jer ne znam da li će me razumeti. Niko ne razume koliko truda ulažem u svakodnevne razgovore, ma koliko oni bili kratki. Plašim se raznih stvari i imam običaj da ruke perem više puta detaljno, da mi cela kuća bude sređena. Da li je ovo stvarno anksioznost ili može biti nešto drugo? Kome da se obratim?
-                            </p> <br>
+                            <p style="text-align: justify; font-weight: normal;"> ' . $pitanje->tekstPitanja . ' </p> 
+                            <br>
                         </div>
-
-                        <!-- DUGMAD -->
-                        <div class="input ">     
-                            <div id="like">
-                                <div>
-                                    <button class="w3-button buttons" onclick="likeFunction(this)"><b><i class="fa fa-thumbs-up"></i> <u onclick="alert('Povećali ste broj lajkova za 1')" style="text-decoration: none; font-weight: normal;">Korisno (3)</u></button> &nbsp
-                                    <button class="w3-button buttons" onclick="likeFunction(this)"><b><i class="fa fa-thumbs-down"></i> <u onclick="alert('Povećali ste broj dislajkova za 1')" style="text-decoration: none; font-weight: normal;">Nije korisno (11)</u></button>
-                                </div>
-
-                                <div style="float: right;">
-                                    <button onclick="window.location.href = 'odgovori.html';" class="w3-button buttons" style="font-weight: normal;"> Pogledaj
-                                        odgovore</button> &nbsp
-                                    <button onclick="window.location.href = 'odgovori.html#text_input';"
-                                            class="w3-button buttons" style="font-weight: normal;">Odgovori</button>
-                                </div>
-                            </div>
-                        </div>
-                        <br>
+                        ';
+
+                        // DUGMAD 
+                        $idPitanja=$pitanje->idPitanje;
+                        $likes=$korisnikOcenioPitanjeModel->findNumOfLikes($idPitanja);
+                        $dislikes=$korisnikOcenioPitanjeModel->findNumOfDislikes(($idPitanja));
+                        $referenca1=site_url("$controller/PostaviLike?pretraga=$idPitanja");
+                        $referenca2=site_url("$controller/PostaviDislike?pretraga=$idPitanja");
+                        $referenca3=site_url("$controller/pregledOdgovora?pretraga=$idPitanja");
```

Comment:
Zašto je tu pretraga=$idPitanja? Id pitanja ovde treba da se prosledi preko URLa. 
Vidi [ovde](https://codeigniter.com/userguide3/general/urls.html#uri-segments) ako hoćeš detaljno, a možda nađeš nešto korisno i [ovde](https://codeigniter.com/userguide3/general/routing.html).
A biće ti najlakše da samo vidiš samo prvi odgovor [na StackOverflowu](https://stackoverflow.com/a/21180270)
```suggestion
                        $referenca3=site_url("$controller/pregledOdgovora/$idPitanja");
```